packages/office-ui-fabric-react/src/components/Toggle/Toggle.types.ts

```diff
@@ -31,6 +31,11 @@ export interface IToggleProps extends React.HTMLAttributes<HTMLElement> {
    */
   label?: string | JSX.Element;
 
+  /**
+   * A name for the toggle's hidden checkbox input.
+   */
+  name?: string;
```

Comment:
be more specific with the param name if it's only going to go on the checkbox

Reply:
I was trying to be consistent with Checkbox and ChoiceGroup. Would you still like this to be different?